Workbooks/AKS/Node Kubelet AKS/Node Kubelet AKS.workbook

```diff
@@ -6,44 +6,9 @@
       "content": {
```

Comment:
Make clusterIdWhereClause required, as it needs to be in place for all queries

Reply:
Make nodes adnd operations required so that the quick query parsing error doesn't show. From the error, it's clear the queries aren't running any faster because they have to wait until the parameter gets a value in order to start executing

Reply:
Make the stepnames for tabs include tab in the name so it is more clear what that step is in the telemetry. E.g. "Overview" becomes "Overview Tab"

Reply:
Revert changes